src/ivalue_manager.rs

```diff
@@ -637,23 +637,56 @@ impl<'a> Manager for RedisIValueJsonKeyManager<'a> {
         }
     }
 
+    ///
+    /// following https://github.com/Diggsey/ijson/issues/23#issuecomment-1377270111
+    ///
     fn get_memory(&self, v: &Self::V) -> Result<usize, RedisError> {
         let res = size_of::<IValue>()
             + match v.type_() {
-                ValueType::Null | ValueType::Bool | ValueType::Number => 0,
+                ValueType::Null | ValueType::Bool => 0,
+                ValueType::Number => {
+                    let num = v.as_number().unwrap();
+                    if num.has_decimal_point() {
+                        // 64bit float
+                        16
+                    } else if num > &INumber::from(-128) && num <= &INumber::from(383) {
```

Comment:
The comment in mentioned ijson issue 23 says
> Else if in the range -128..=383 (none)

Isn't it inclusive? Should be `else if num >= &INumber::from(-128) && ...` ?

Reply:
@Diggsey ?